packages/local-environment/src/network.ts

```diff
@@ -175,19 +189,20 @@ export class HardhatNetwork extends Network {
   keys: Key[];
   handler: DockerizedNetworkActor;
 
-  constructor(name: string, domain: number, options?: HardhatNetworkOptions) {
-    super(name, domain, options?.chainId || domain);
-    this.handler = options?.handler || new DockerizedNetworkActor(this.name);
+  constructor(name: string, domain: number, docker?: Dockerode, options?: HardhatNetworkOptions) {
+    super(name, domain, domain, docker);
```

Comment:
I noticed we're passing in `domain` twice, is that right? Before it looks like we passed in `options?.chainId || domain` for the third parameter of super

Reply:
Fixed this just now